README.md

```diff
@@ -176,6 +176,36 @@ pytest <path-to-triton-shared>/python/examples
 ```
 In addition to testing on the tutorial kernels, there are many lit tests covering various scenarios.
 
+## Intermediate Representation (IR) Dumps
+
+To facilitate debugging and analysis, the Triton Shared project now supports emitting all intermediate representations (IRs) generated during the compilation process. This functionality is controlled via the environment variable `TRITON_SHARED_DUMP_PATH`.
+
```

Comment:
nit: i would change Triton Shared to just triton-shared. Otherwise looks good to me. Thank you.

Reply:
no problem, updated, thank you!